src/i18n/en-system-messages.ts

```diff
@@ -1,21 +1,21 @@
-      ${SystemMessagesName.MotionHasPassed} {{motionTag} has {passedTag} and may be finalized.}
-      ${SystemMessagesName.MotionRevealPhase} {It's time to {revealTag} votes to the world!}
-      ${SystemMessagesName.MotionHasFailedNotFinalizable} {{motionTag} has failed.}
-      ${SystemMessagesName.MotionHasFailedFinalizable} {{motionTag} has {failedTag} and may be finalized.}
-      ${SystemMessagesName.MotionVotingPhase} {{votingTag} has started! Voting is secret 🤫, and weighted by Reputation.}
-      ${SystemMessagesName.MotionFullyStaked} {{motionTag} is fully staked; Staking period has reset. As long as there is no {objectionTag} , the motion will pass.}
-      ${SystemMessagesName.MotionFullyStakedAfterObjection} {{motionTag} is fully staked.}
-      ${SystemMessagesName.ObjectionFullyStaked} {{objectionTag} is fully staked; Staking period has reset. If the {motionTag} gets fully staked, a vote will start immediately; if not, the {motionTag} will fail.}
-      ${SystemMessagesName.MotionRevealResultObjectionWon} {{motionTag} failed. {voteResultsWidget} The motion will fail at the end of the escalation period unless the dispute is escalated to a higher team.}
-      ${SystemMessagesName.MotionRevealResultMotionWon} {{motionTag} passed! {voteResultsWidget} The motion will pass at the end of the escalation period unless the dispute is escalated to a higher team.}
-      ${SystemMessagesName.MotionCanBeEscalated} {{escalateTag} period started. {spaceBreak}{spaceBreak} If you believe this result was unfair, and would be different if more people were involved, you may escalate it to a higher team.}
+  'systemMessage.title': `{eventName, select,
+      ${SystemMessages.EnoughExitRecoveryApprovals} {Enough permission holders have now signed to exit recovery mode. As long as no further storage slots are updated, a recovery permission holder may now sign a transaction to reactivate the colony.}
+      ${SystemMessages.MotionHasPassed} {{motionTag} has {passedTag} and may be finalized.}
+      ${SystemMessages.MotionRevealPhase} {It's time to {revealTag} votes to the world!}
+      ${SystemMessages.MotionHasFailedNotFinalizable} {{motionTag} has failed.}
+      ${SystemMessages.MotionHasFailedFinalizable} {{motionTag} has {failedTag} and may be finalized.}
+      ${SystemMessages.MotionVotingPhase} {{votingTag} has started! Voting is secret 🤫, and weighted by Reputation.}
+      ${SystemMessages.MotionFullyStaked} {{motionTag} is fully staked; Staking period has reset. As long as there is no {objectionTag} , the motion will pass.}
+      ${SystemMessages.MotionFullyStakedAfterObjection} {{motionTag} is fully staked.}
+      ${SystemMessages.ObjectionFullyStaked} {{objectionTag} is fully staked; Staking period has reset. If the {motionTag} gets fully staked, a vote will start immediately; if not, the {motionTag} will fail.}
```

Comment:
This one shows up even if the motion is already fully staked in favour, whereas in the dapp it doesn't show in this case, since it goes straight to voting and shows that message.

Reply:
Made a change in the block ingestor to only store the ObjectionFullyStaked message if the Nay vote is greater than the Yay vote when the block ingestor receives a fully staked nay vote.